src/schema.json

```diff
@@ -0,0 +1,405 @@
+{
+	"definitions": {
+		"mode": {
+			"type": "string",
+			"enum": [
+				"dist",
+				"dev",
+				"unit",
+				"functional"
+			]
+		},
+		"watch": {
```

Comment:
This isn't correct any more, it is a boolean.

Reply:
:+1: